sources/packages/api/src/services/student/student.service.ts

```diff
@@ -176,6 +176,21 @@ export class StudentService extends RecordDataModelService<Student> {
     return this.repo.find({ validSIN: null });
   }
 
+  /**
+   * Gets a student's valid sin status
+   * @returns Students SIN validation status.
+   * @param userId information needed to select the user.
+   */
+  async getStudentSinStatus(userId: number): Promise<Student> {
+    const sinStatus = await this.repo
```

Comment:
you can directly return the await statement here. It will clear the sonar code smell that we got.